suzieq/shared/utils.py

```diff
@@ -1229,3 +1229,29 @@ def log_suzieq_info(name: str, c_logger: logging.Logger = None,
             'higher with 4 cores and 16 GB RAM')
     if prev_level > logging.INFO:
         c_logger.setLevel(prev_level)
+
+
+def set_rest_engine(cfg: Dict[str, Any]) -> Tuple:
```

Comment:
type hints can be improved
```suggestion
def set_rest_engine(cfg: Dict[str, Any]) -> Tuple[str, str, str, str]:
```

Reply:
fixed